app/views/users/show.html.erb

```diff
@@ -0,0 +1,24 @@
+
+<h1>User Listing</h1>
+
+<table class="table table-striped">
+  <thead class="thead-dark">
+    <tr>
+      <th scope="col">Email Address</th>
+      <th scope="col">First Name</th>
+      <th scope="col">Last Name</th>
+      <th scope="col">Role</th>
+      <th scope="col">Edit</th>
+    </tr>
+  </thead>
+         
+    <tr>
+        <td class="p-2"><%= @user.email %></td>
+        <td class="p-2"><%= @user.full_name.split(" ").first %></td>
+        <td class="p-2"><%= @user.full_name.split(" ").last %></td>
+        <td class="p-2"><%= Role.find_by(id: @user.role).name %></td>
```

Comment:
You can leverage power of `ActiveRecord` associations here and write this line like this
`@user.roles.pluck(:name)`